extensions/BMO/template/en/default/bug/create/create-firefox-security.html.tmpl

```diff
@@ -0,0 +1,188 @@
+
+@media screen and (max-width: 800px) {
+  #header {
+    display: none;
+  }
+}
+
+#firefox_security_form input[type="text"],
+#firefox_security_form textarea {
+  width: 100%;
+}
+
+#firefox_security_form input[type="submit"] {
+  float: right;
+}
+
+#comment {
+  width: 100% !important;
+  display: block;
+  width: 100%;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+#bugzilla-body {
+    margin-left: auto;
+    margin-right: auto;
+    background-color: white;
+    max-width: 1024px;
+}
+
+#bugzilla-body h1 {
+  text-align: center;
+  padding-top: 2px;
+  padding-bottom: 0px;
+  margin-bottom: 0px;
+}
+
+#bugzilla-body .head_desc {
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 0px;
+  padding-top: 0px;
+}
+
+#firefox_security_form {
+  padding: 10px;
+}
+
+#firefox_security_form .required:after {
+  content: " *";
+  color: red;
+}
+#firefox_security_form .field_label {
+  font-weight: bold;
+}
+#firefox_security_form .field_desc {
+  padding-bottom: 3px;
+}
+#firefox_security_form .field_desc,
+#firefox_security_form .head_desc {
+  word-wrap: normal;
+}
+#firefox_security_form .head_desc {
+  padding-top: 5px;
+  padding-bottom: 12px;
+}
+#firefox_security_form .form_section {
+  margin-bottom: 10px;
+}
+#useful-links {
+  display: none;
+}
+[% END %]
+
+[% inline_javascript = BLOCK %]
+function validateAndSubmit() {
+  var alert_text = '';
+  if (!isFilledOut('short_desc')) alert_text += 'Please enter a value for summary.\n';
+  if (!isFilledOut('comment')) alert_text += 'Please enter a value for comment.\n';
+  if (alert_text != '') {
+    alert(alert_text);
+    return false;
+  }
+  return true;
+}
+[% END %]
+
+[% PROCESS global/header.html.tmpl
+   title = "Firefox Security Form"
+   generate_api_token = 1
+   style = inline_style
+   javascript = inline_javascript
+   javascript_urls = [ 'extensions/BMO/web/js/form_validate.js',
+                       'extensions/BMO/web/js/attach-desc.js',
+                       'js/field.js', 'js/util.js' ]
+%]
+
+[% USE Bugzilla %]
+[% cgi = Bugzilla.cgi %]
+
+<h1>Firefox Security Form</h1>
+
+<form id="firefox_security_form" method="post" action="post_bug.cgi" enctype="multipart/form-data"
+      onSubmit="return validateAndSubmit();">
+  <input type="hidden" name="product" value="Firefox">
+  <input type="hidden" name="component" value="Security">
+  <input type="hidden" name="rep_platform" value="unspecified">
+  <input type="hidden" name="op_sys" value="unspecified">
+  <input type="hidden" name="version" value="unspecified">
+  <input type="hidden" name="bug_severity" id="bug_severity" value="normal">
+  <input type="hidden" name="priority" id="priority" value="--">
+  <input type="hidden" name="target_milestone" id="target_milestone" value="---">
+  <input type="hidden" name="status_whiteboard" id="status_whiteboard"
+         value="[reporter-external] [web-bounty-form] [verif?]">
```

Comment:
I'd suggest  [client-bounty-form] instead of [web-bounty-form]

Reply:
I agree.